detox/test/package.json

```diff
@@ -39,7 +39,7 @@
   "devDependencies": {
     "@babel/core": "^7.12.9",
     "@babel/runtime": "^7.12.5",
-    "@types/node": "^14.14.20",
+    "@types/node": "^12.20.37",
```

Comment:
why do we need to downgrade the requirement?

Reply:
Because Detox supports Node.js 12 minimally. That was erroneous to orientate ourselves on types that might be not available in minimally claimed version to be supported by Detox.